modules/couchbase/src/test/java/org/testcontainers/couchbase/Couchbase6_0Test.java

```diff
@@ -0,0 +1,14 @@
+package org.testcontainers.couchbase;
+
+import org.junit.Rule;
+
+public class Couchbase6_0Test extends BaseCouchbaseContainerFtsTest {
```

Comment:
is it really needed? Can FTS be tested with old versions, or this is a feature of 6.0?

Reply:
Yes, FTS is available begining from version 6.0

Reply:
Ok. But let's not test every version, 6.0 is enough

Reply:
Ok, I prefer to test the latest then. I'll remove 6.0 and keep 6.5